### ISSUE TITLE
Autoantibodies in rare hemolytic diseases with coagulation deficiencies

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#10286]
[20240119 MHEMO.xlsx](https://github.com/obophenotype/human-phenotype-ontology/files/13988798/20240119.MHEMO.xlsx)
Hello Peter,
Is this Excel file fine for you?
Please let me know,
Best wishes for the New Year!
Maud

**Comment by pnrobinson** [comment]
@MaudDLVT 
thanks again so much. The excel file looks great, we will add the terms in the next week.

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000492
- label: Anti-factor V antibody positivity
- parents: [Anti-coagulation factor antibody positivity (HP:6000351)](https://hpo.jax.org/app/browse/term/HP:6000351)
- pmid: PMID:33505468
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against factor V.
- comment: 
- orcid: orcid:0000-0002-3387-1838
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10286
- synonyms: Anti-FV antibody positivity(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000493
- label: Anti-factor VII antibody positivity
- parents: [Anti-coagulation factor antibody positivity (HP:6000351)](https://hpo.jax.org/app/browse/term/HP:6000351)
- pmid: PMID:26962982
- def: The presence of autoantibodies (immunoglobulins) in the serum that react against factor VIII.
- comment: 
- orcid: orcid:0000-0002-3387-1839
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10286
- synonyms: Anti-FVII antibody positivity(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000494
- label: Anti-factor VIII antibody positivity
- parents: [Anti-coagulation factor antibody positivity (HP:6000351)](https://hpo.jax.org/app/browse/term/HP:6000351)
- pmid: PMID:3116540
- def: The presence of autoantibodies (immunoglobulins) in the serum that react against factor VIII.
- comment: 
- orcid: orcid:0000-0002-3387-1840
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10286
- synonyms: Anti-FVIII antibody positivity(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000495
- label: Anti-factor IX antibody positivity
- parents: [Anti-coagulation factor antibody positivity (HP:6000351)](https://hpo.jax.org/app/browse/term/HP:6000351)
- pmid: PMID:28594432|PMID:27501440
- def: The presence of autoantibodies (immunoglobulins) in the serum that react against factor IX.
- comment: 
- orcid: orcid:0000-0002-3387-1841
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10286
- synonyms: Anti-FIX antibody positivity(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000496
- label: Anti-factor X antibody positivity
- parents: [Anti-coagulation factor antibody positivity (HP:6000351)](https://hpo.jax.org/app/browse/term/HP:6000351)
- pmid: PMID:24074948
- def: The presence of an antibody in the blood circulation that is directed against factor X.
- comment: 
- orcid: orcid:0000-0002-3387-1836
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10286
- synonyms: Anti-FX antibody positivity(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000498
- label: Anti-factor XIII antibody positivity
- parents: [Anti-coagulation factor antibody positivity (HP:6000351)](https://hpo.jax.org/app/browse/term/HP:6000351)
- pmid: PMID:32759568|PMID:29460500
- def: The presence of an antibody in the blood circulation that is directed against factor XIII.
- comment: 
- orcid: orcid:0000-0002-3387-1836
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10286
- synonyms: Anti-FXIII antibody positivity(NONE)
